hack/vmimage.sh

```diff
@@ -3,12 +3,20 @@
 cleanup() {
   set +e
 
+  if [[ -e /var/run/secrets/kubernetes.io ]] && [ "${JOB_TYPE} == "periodic" ]; then
+    if [ $PHASE == "build_complete" ]; then
```

Comment:
In traps we can check exit code with with `$?` ;)

Reply:
Ah, right! We can make it work somehow like this:

```bash
    exit_code=$?
    set +e
```

But I'm not sure if the efforts worth it.